firmware/usb_test/README

```diff
@@ -7,7 +7,7 @@ device should be detected.  For example (on Linux):
 [134437.193956] usb 8-1.2: New USB device found, idVendor=ffff, idProduct=0005
 [134437.193962] usb 8-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
 [134437.193969] usb 8-1.2: Product: Ubertooth serial
-[134437.193974] usb 8-1.2: Manufacturer: http://ubertooth.sourceforge.net/
+[134437.193974] usb 8-1.2: Manufacturer: http://github.com/greatscottgadgets/ubertooth
```

Comment:
Should we update this line to "https" instead?

Reply:
It's the actual output from the firmware on a device I tested, but actually it looks like our latest manufacturer strings is "Great Scott Gadgets". I'll change it to that.